src/main/plugins/org.dita.html5/xsl/task.xsl

```diff
@@ -164,13 +167,65 @@ See the accompanying LICENSE file for applicable license.
     </section>
   </xsl:template>
   
+  <!-- Nested steps: no section container, no label, no special format for single step -->
+  <xsl:template match="*[contains(@class,' task/step ')]
+    /*[contains(@class,' task/steps ') or contains(@class,' task/steps-unordered ')]"
+    mode="common-processing-within-steps">
+    <xsl:param name="step_expand"/>
+    <xsl:param name="list-type">
+      <xsl:choose>
+        <xsl:when test="contains(@class,' task/steps ')">ol</xsl:when>
+        <xsl:otherwise>ul</xsl:otherwise>
+      </xsl:choose>
+    </xsl:param>
```

Comment:
Even though this mostly a copy of the other template, we should move to XSLT 2.0 code when possible

```suggestion
    <xsl:param name="list-type" select="if (contains(@class,' task/steps ')) then 'ol' else 'ul'"/>
```